config/matrix-api.yml

```diff
@@ -56,6 +56,22 @@ definitions:
     properties:
       message:
         type: string
+  DssNotification:
```

Comment:
As mentioned in slack, I was surprised to not see the logic for creating subscriptions in this PR. I think we should be deleting and recreating the notifications as part of the deployment flow with the queries explicitly defined in the code? Perhaps have a NotificationRegister (terrible name, pick something else) class where this is defined

Reply:
+1 on version controlling the query and recreating it per deployment. Work captured [here](https://app.zenhub.com/workspaces/dcp-backlogs-5ac7bcf9465cb172b77760d9/issues/humancellatlas/matrix-service/224)